validator_derive/src/lib.rs

```diff
@@ -178,6 +179,19 @@ impl ToTokens for ValidateField {
             quote!()
         };
 
+        // Must not match validation
+        let must_not_match = if let Some(must_not_match) = self.must_not_match.clone() {
+            // TODO: handle option for other
```

Comment:
Is that still a TODO?

Reply:
As I understood correctly - it was handled